pyfstat/make_sfts.py

```diff
@@ -484,6 +509,11 @@ def _basic_setup(self):
         if self.tref is None:
             self.tref = self.tstart
 
+        if getattr(self, "SFTWindowBeta", None):
+            logger.warning(
+                "Option 'SFTWindowBeta' is defunct, please use 'SFTWindowParam'."
+            )
```

Comment:
Should this raise an exception? Right now setting `SFTWindowBeta` does *nothing*, doesn't it?

It should either set `SFTWindowParam` or raise exception.

Reply:
Good point, I think it might fail with a lalpulsar error further on and the warning is just for clarity, but I'll have to check to remind myself of what actually happens.

Reply:
Indeed it was not failing in any obvious way when passed the old name. Changed to an error in b0cd43b07e99d8ba5239770a60a1c162eb52207f